ambry-rest/src/main/java/com.github.ambry.rest/RestServerMain.java

```diff
@@ -80,10 +81,10 @@ public static void main(String[] args) {
    * @throws IOException
    */
   private static SSLFactory getSSLFactoryIfRequired(VerifiableProperties verifiableProperties) throws Exception {
-    ClusterMapConfig clusterMapConfig = new ClusterMapConfig(verifiableProperties);
-    if (clusterMapConfig.clusterMapHttp2NetworkClientEnabled) {
+    if (new RouterConfig(verifiableProperties).routerEnableHttp2NetworkClient) {
```

Comment:
We should change this to instantiate the SslFactory within NetworkClientFactory, since there can be some incompatibilities between the HTTP2 factory and the one used for RestServer.